static-location.js

```diff
@@ -1,17 +1,4 @@
 // Generates static `useLocation` hook. The hook always
 // responds with initial path provided.
 // You can use this for server-side rendering.
-export default (path = "/", { record = false } = {}) => {
-  let hook;
-  const navigate = (to, { replace } = {}) => {
-    if (record) {
-      if (replace) {
-        hook.history.pop();
-      }
-      hook.history.push(to);
-    }
-  };
-  hook = () => [path, navigate];
-  hook.history = [path];
```

Comment:
We need to figure out how to add support for recording changes back. I know that the signature of `useLocation` is different now, but there should be a way for people to have proper testing support. See https://github.com/molefrog/wouter/issues/113